common-rest/src/test/java/org/uniprot/api/rest/validation/ValidIdsDownloadRequestTest.java

```diff
@@ -0,0 +1,194 @@
+package org.uniprot.api.rest.validation;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.validation.ConstraintValidatorContext;
+
+import org.hibernate.validator.internal.engine.constraintvalidation.ConstraintValidatorContextImpl;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.uniprot.store.config.UniProtDataType;
+
+/**
+ * @author sahmad
+ * @created 26/07/2021
+ */
+class ValidIdsDownloadRequestTest {
```

Comment:
If you agree to change the validator name, make sure you also rename the test class